subgraphs/_reference_/src/sdk/protocols/bridge/tokens.ts

```diff
@@ -37,7 +37,7 @@ export class TokenManager {
     this.presaver = presaver;
   }
 
-  getOrCreateToken(address: Address): Token {
+  getOrCreateToken(address: Bytes): Token {
```

Comment:
I think token can be left as Address always, because it will always refer to a token in the chain where the subgraph is running, which will always be EVM. Right?

Same thing on the initializer interface.

Reply:
I changed this because I think `pool.getDestinationTokenRoute(crossToken)` passed to `account.transferOut()` needs to support non-EVM addresses.

```
  const crossToken = sdk.Tokens.getOrCreateCrosschainToken(
    dstChainId,
    dstPoolId,
    crosschainTokenType,
    token
  );
  pool.addDestinationToken(crossToken);
  const txHash = event ? event.transaction.hash : call!.transaction.hash;
  const acc = sdk.Accounts.loadAccount(sender);
  const transfer = acc.transferOut(
    pool,
    pool.getDestinationTokenRoute(crossToken)!,
    receiver,
    amount,
    txHash
  );
```

Reply:
How does `getOrCreateToken(Address)` affect `getDestinationTokenRoute` or `acc.transferOut`?

Reply:
You're right. token doesn't need to be Bytes type in this case. I was thinking that the token argument in the `sdk.Tokens.getOrCreateCrosschainToken` might be a non-EVM token address, which should not happen.